ods_ci/build/Dockerfile

```diff
@@ -60,9 +60,10 @@ RUN  chmod +x ods_ci/build/run.sh &&\
 COPY pyproject.toml .
 COPY poetry.lock .
 COPY README.md .
+RUN alternatives --install /usr/local/bin/python3 python3 /usr/bin/python${PYTHON_VERSION} 1
```

Comment:
This line should be the only functional change in the PR. It looks ok to me in principle. It can break things, for example we will not have `python3-distro` installed on python3.11, so if there's something that needs that package, it will break. But overall, feels reasonably safe.

Reply:
ensure to run an entire Smoke job before merging it.

About distro, https://github.com/red-hat-data-services/ods-ci/pull/817

Reply:
> closing this one because it's not high in the priority queue, besides this is not causing issues.

hopefully this will remain true